IcemoonUnderground_Above.dmm

```diff
@@ -0,0 +1,732 @@
+"aa" = (/obj/docking_port/stationary{dir = 8; dwidth = 2; height = 5; id = "laborcamp_away"; name = "labor camp"; width = 9},/turf/closed/mineral/random/snow,/area/icemoon/underground/unexplored/rivers)

```

Comment:
delete this file

Reply:
Why delete this file? This file is needed to give winter coat to gulag.

Reply:
note the file location and the fact you have 2 same-named files in the files changed tab.